content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -336,7 +336,7 @@ kubectl logs -f my-pod -c my-container              # stream pod container logs
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox:1.28 -- sh  # Run pod as interactive shell
 kubectl run nginx --image=nginx -n mynamespace      # Start a single instance of nginx pod in the namespace of mynamespace
-kubectl run nginx --image=nginx                     # Run pod nginx and write its spec into a file called pod.yaml
+kubectl run nginx --image=nginx                     # Generate spec for running pod nginx and write it into a file called pod.yaml
 --dry-run=client -o yaml > pod.yaml
```

Comment:
I think this should be:
```suggestion
kubectl run nginx --image=nginx \
  --dry-run=server -o yaml > pod.yaml               # Generate spec for running pod nginx and write it
                                                    # into a file called pod.yaml
```

Reply:
I agree with this suggestion
@swadhinbharat , ptal at sftim's [suggestion](https://github.com/kubernetes/website/pull/35732#discussion_r938995981)

Reply:
Hi @sftim 

I am not sure if the "--dry-run=server" fits in this context ... the simplicity of all other commands listed in this section make me incline towards using "--dry-run=client" for generating the template as an alternative to the imperative statement. The server dry run seems like an overkill. Or rather both the statements should be added to the section.

Reply:
I have a feeling that client-side dry run is deprecated. Have I got that right?

Reply:
@sftim I am not aware of Dry Run for Client being deprecated. Just the syntax has changed to explicitly specify now with --dry-run=client since the --dry-run=true / --dry-run is deprecated. I was hoping someone else also can vouch my understanding. There was no such statements anywhere as well stating this has been deprecated.